gene-network/src/main/resources/js/gn.js

```diff
@@ -16,7 +16,7 @@ $(function () {
             },
             validExtensions: ['.vcf', '.vcf.gz', '.tsv', '.tsv.gz', '.txt', '.txt.gz', '.tab', 'tab.gz'],
             showNameFieldExtensions: ['.vcf', '.vcf.gz', '.tsv', '.tsv.gz', '.txt', '.txt.gz', '.tab', 'tab.gz'],
-            maxFileSizeMB: 3
+            maxFileSizeMB: 50
```

Comment:
I'll add a comment, but it remains a bit arbitrary. It's an estimate of the filesize of an exome VCF